src/devices/GrovePi/Models/GrovePiCommands.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Iot.Device.GrovePiDevice.Models
+{
+    /// <summary>
+    /// GroovePi commands to read, write, setup pins and access special sensors
+    /// Note that those commands are supports in most of the recent firmware with version higher than 1.2.1
+    /// </summary>
+    public enum GrovePiCommands
```

Comment:
should this be private?

Reply:
No as if you want to implement your own sensors, you may need it